spec/registry.md

```diff
@@ -619,6 +619,23 @@ All other values produce an _Invalid Expression_ error.
 > or the type `com.ibm.icu.util.CurrencyAmount` can be used to set the currency and related
 > options (such as the number of fraction digits).
 
+### Non-Negative Integer Options
+
+Some _options_ of number _functions_ are defined to take a "non-negative integer".
+Implementations of number _functions_ use these _options_ to control aspects of numeric display
+such as the number of fraction, integer, or significant digits.
+
+A "non-negative integer" is an _option_ value that the _function_ evaluates as an integer 
+greater than or equal to zero.
+Implementations MAY define an upper limit on the resolved value 
```

Comment:
Do we want to say anything about what implementations should do about the lower bound?

Reply:
The lower bound is always zero.

Reply:
Well, yes :) But it seems like we don't say anything about what should happen when the passed value is, in fact, negative? Should functions emit an error? Perhaps we need a sister error to the `Operand Mismatch Error`, but for invalid options?

Should I file a separate issue about this?

Reply:
`Invalid Option` suggests itself as a sub-species of `Invalid Expression`.

A negative number won't parse correctly about the same way that the string `moo` or the string `$57%` does not parse as a non-negative integer. FWIW, `1,234` does not parse according to the rules laid out here nor does `11.0`

The reason for upper bounds, of course, is overflow prevention.